projects/plugins/social/tests/e2e/flows/connection.js

```diff
@@ -11,9 +11,9 @@ export async function connect( page, premium = false ) {
 	socialPage = await JetpackSocialPage.init( page );
 
 	if ( premium ) {
-		socialPage.getSocial();
+		await socialPage.getSocial();
 		// todo add purchase steps
 	} else {
-		socialPage.startForFree();
+		await socialPage.startForFree();
```

Comment:
This unawaited promise misguded me and wasted at least an hour to find the actual cause of the failing E2E tests. Here the promises got fired and the test runner continued running the tests without awaiting here and when it failed here. It showed the error at the place where the test was at the moment - the editor.